core/src/tasks/schedule/enqueueRuns.ts

```diff
@@ -28,9 +33,10 @@ export class ScheduleEnqueueRuns extends CLSTask {
     const runIfNotRecurring =
       params.runIfNotRecurring === undefined ? false : params.runIfNotRecurring;
 
-    const schedules = await Schedule.scope(null).findAll({
-      where: { state: "ready" },
-    });
+    const where: WhereOptions<Schedule> = { state: "ready" };
+    if (params.scheduleIds) where.id = params.scheduleIds;
+
+    const schedules = await Schedule.scope(null).findAll({ where });
```

Comment:
The default scope of a schedule is state=ready.  https://github.com/grouparoo/grouparoo/blob/8bca2324f6ba8b093afc95a7350a9c6f89c001c6/core/src/models/Schedule.ts#L80-L82


We probably can skip that part of the where clause and the scope(null)

Reply:
👍🏼 done in 7084a51

There's already a test in `enqueueRuns` that checks that drafts aren't enqueued and it seems to still be passing after the change